libs/deps/merch-card.js

```diff
@@ -1,7 +1,7 @@
-// branch: main commit: 258ad3d851cc4945eae2aab2dcc80a8d0c14d861 Fri, 28 Jun 2024 14:19:21 GMT
-import{html as a,LitElement as j}from"/libs/deps/lit-all.min.js";import{LitElement as q,html as T,css as F}from"/libs/deps/lit-all.min.js";var s=class extends q{static properties={size:{type:String,attribute:!0},src:{type:String,attribute:!0},alt:{type:String,attribute:!0},href:{type:String,attribute:!0}};constructor(){super(),this.size="m",this.alt=""}render(){let{href:e}=this;return e?T`<a href="${e}">
+// branch: MWPW-154026-cta-long-text commit: b9a46be0956fd4df485c08a14cb59a0e51814c78 Fri, 12 Jul 2024 14:20:01 GMT
```

Comment:
@npeltier , @yesil , @3ch023 - what's the correct way to build this dependency? I've seen it being built from master, develop, some feature branches along with multiple comments saying it should be built from develop or from a feature branch. Can you please shed some light here? Thanks!

Reply:
It should be built from main